pkg/controller/pod.go

```diff
@@ -591,7 +591,9 @@ func (c *Controller) handleDeletePod(pod *v1.Pod) error {
 				return err
```

Comment:
should also skip if subnet not exist
https://github.com/kubeovn/kube-ovn/pull/1335/commits/086ca7616a4ce86ccfef20f221f009c1a3d4013c#diff-449e8d1bd46b9c978e208dde912a1d7a76d17ce553067f8d0a67a27f17338d26L590